src/main/java/Duke/Ui.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * This class deals with the interactions with the user and prints
+ * the necessary messages according to the user input
+ */
+package Duke;
+
+import Tasks.Task;
+
+import java.util.Scanner;
+
+public class Ui {
+    private Scanner scanner;
+
+    /**
+     * Initialises a new UI with a scanner to get user input
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Returns the command input by the user
+     *
+     * @return string containing the user input per line
+     * @throws DukeException which shows an error if there is no more lines
+     *         left in the user input
+     */
+    public String readCommand() throws DukeException {
+        if (scanner.hasNextLine())
+            return scanner.nextLine();
+        else {
+            throw new DukeException(Constants.NO_MORE_LINES);
+        }
+    }
```

Comment:
This works yay, but can also consider try-catch cos is normally used for error handling

Reply:
okay! Thanks for the suggestion :)